pkg/controller/bundle/bundle_unpacker_test.go

```diff
@@ -789,7 +1313,8 @@ func TestConfigMapUnpacker(t *testing.T) {
 			)
 			require.NoError(t, err)
 
-			res, err := unpacker.UnpackBundle(tt.args.lookup)
+			// The annotation unpack timeout arg is negative so it is ignored for these tests
+			res, err := unpacker.UnpackBundle(tt.args.lookup, -1*time.Minute)
```

Comment:
I would have expected at least one test where the timeout is varied and its inclusion on the job spec is verified